src/pages/webhooks/release-notes.md

```diff
@@ -9,6 +9,28 @@ keywords:
 
 These release notes describe the latest version of Adobe Commerce Webhooks.
 
+## Version 1.1.0
+
+### Release date
+
+November 30, 2023
+
+### Enhancements
+
+* You can now define [conditional webhooks](conditional-webhooks.md) that run only when configured conditions are met. <!--- CEXT-2541 -->
+
+* Prior to this release, the only supported webhook response values were `success` and `exception`. This release adds support for `add`, `remove`, and `replace` [responses](responses.md#responses). <!--- CEXT-2405, 2413, 2468 -->
+
+* You can now [configure specific fields within an array](hooks.md#configure-hooks) to transmit in a webhook. <!--- CEXT-2521 -->
```

Comment:
I would skip this one from the release notes as it was an improvement for the new `add` operation